corehq/apps/app_manager/models.py

```diff
@@ -4467,6 +4514,14 @@ def get_hierarchy(case_type):
 
         return meta
 
+    def get_subcase_types_of_module(self, module):
+        """
+        Return the subcase types defined across an app for the case type of a module
+        """
+        return {i for m in self.get_modules()
+                if m.case_type == module.case_type
+                for i in m.get_subcase_types()}
```

Comment:
`i` is a misleading variable name since it usually refers to a loop iteration count.